report/clients/treemap.js

```diff
@@ -13,6 +13,7 @@ import {I18n} from '../renderer/i18n.js';
 import {Logger} from '../renderer/logger.js';
 import {TextEncoding} from '../renderer/text-encoding.js';
 
+/** @typedef {import('../../lighthouse-treemap/types/treemap')} */
```

Comment:
was pleasantly surprised this worked

EDIT: except it works too well.... this pollutes the `window` _everywhere_ in lighthouse-core,cli. oops. @brendankenny any ideas, or just ts-expect-error? These files are rather temporary if the report refactor works out...